iOSSampleAppWithSDK/Podfile.lock

```diff
@@ -73,12 +73,11 @@ SPEC REPOS:
 
 EXTERNAL SOURCES:
   LocusLotrSdk:
-    :branch: frameWorkUpdate
     :git: "git@github.com:locus-taxy/lotr-sdk-ios.git"
 
 CHECKOUT OPTIONS:
   LocusLotrSdk:
-    :commit: a458084b656b462386b999f922507322a1ee5aa3
+    :commit: ffe228fec325bb373eca2e6560ceccdbaba1b41f
```

Comment:
I think we should remove the commit. It will by default pick up the latest commit from the master branch IMO.

Reply:
this happens only when he runs the pod install on the project